Ref: Update Flutter and Dart docs to reflect beta version (#2733)
getsentry/sentry-docs: src/platforms/flutter/usage/advanced-usage.mdx

## Known limitations

> - Flutter `split-debug-info` and `obfuscate` flag isn't supported on iOS yet, but only on Android, if this feature is enabled, Dart stack traces are not useful

Author: ```suggestion
- Flutter `split-debug-info` and `obfuscate` flag aren't supported on iOS yet, but only on Android, if this feature is enabled, Dart stack traces are not human readable
```

> ## Known limitations

Reviewer: ```suggestion
## Known Limitations
```

Reviewer: Use Title Case for titles.

> - Flutter `split-debug-info` and `obfuscate` flag aren't supported on iOS yet, but only on Android, if this feature is enabled, Dart stack traces are not human readable

Reviewer: ```suggestion
- Flutter `split-debug-info` and `obfuscate` flag aren't yet supported on iOS; they are supported only on Android. If this feature is enabled, Dart stack traces are not human readable.
```

- If you enable the `split-debug-info` feature, you must upload the Debug Symbols manually.

> ## Tips for catching errors

Reviewer: Would you rather this content be on a Troubleshooting page? (similar to https://docs.sentry.io/platforms/javascript/troubleshooting/)

```suggestion
## Tips for Catching Errors
```
